.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Question
+    url: https://github.com/zowe/perf-timing/discussions
```

Comment:
I know this repo is likely to go away, but curious if we should point people directly to the Q&A section